src/views/tls/tls-version/index.js

```diff
@@ -4,45 +4,88 @@ import {CardContent} from '@material-ui/core';
 import CardHeader from '@material-ui/core/CardHeader';
 import ChartContainer from 'common/chart-container';
 import CircularProgress from '@material-ui/core/CircularProgress';
+import MenuItem from '@material-ui/core/MenuItem';
 import MuiAlert from '@material-ui/lab/Alert';
 import Paper from '@material-ui/core/Paper';
 import {ResponsivePie} from '@nivo/pie';
 import Snackbar from '@material-ui/core/Snackbar';
 import Tab from '@material-ui/core/Tab';
 import Tabs from '@material-ui/core/Tabs';
 import React, {useEffect, useState} from 'react';
+// eslint-disable-next-line no-use-before-define
+import Select from '@material-ui/core/Select';
```

Comment:
Also, you got an error in the automated checks. Install the ESLint extension in VSCode, so it can highlight this kind of errors before you push the code

Reply:
Fixed